packages/web-config-server/src/apiV1/utils/fetchIndicatorValues/checkAllDataElementsAreDhisIndicators.js

```diff
@@ -9,14 +9,13 @@
 export const checkAllDataElementsAreDhisIndicators = async (models, dataElementCodes) => {
   const dataElements = await models.dataSource.find({
     code: dataElementCodes,
-    type: 'dataElement',
```

Comment:
Any reason for removing this? I would have thought some codes could be duplicated between data elements and data groups, so better to be safe?

Reply:
Sorry I think that was a mistake, will reverted!